File: Backend/Inzynierka/Models/UrbanObject.cs
using NetTopologySuite.Geometries;

namespace Inzynierka
{
    public record UrbanObject(int Id)
    {
        public string Name { get; set; } = default!;
        public string Type { get; set;  } = default!;
        public Geometry Geom { get; set;  } = default!;
        public int District { get; set; } = default!;

    }
}

File: Frontend/src/app/models/UrbanObject.ts
import { UrbanObjectType } from "./UrbanObjectType";
import { Coordinate } from 'ol/coordinate';

export interface UrbanObject {
    id: number;
    name: string;
    type: UrbanObjectType;
    geom: GeomModel;
    district: number;
}

export interface GeomModel {
    type: string,
    coordinates: Coordinate | Coordinate[] | Coordinate[][]
}